internal/ingress/controller/nginx.go

```diff
@@ -991,30 +991,47 @@ func configureBackends(rawBackends []*ingress.Backend) error {
 	return nil
 }
 
+type server struct {
+	Hostname    string `json:"hostname"`
+	Certificate string `json:"certificate"`
+}
```

Comment:
I don't see how is this being used

Reply:
Removed. I defined this before the sslConfiguration struct.